components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -151,6 +159,19 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
         let wrappedComponent: React.ReactNode;
 
         if (isDraggable) {
+            let selectedCount: React.ReactNode | undefined;
```

Comment:
I think the `| undefined` might be unnecessary. I think ReactNode is nullable on its own